sw/device/lib/base/memory.c

```diff
@@ -6,11 +6,18 @@
 
 #include "sw/device/lib/base/macros.h"
 
+#ifdef OT_PREFIX_MEM_FUNCTIONS
+#define MANGLE(name) ot_##name
+#else
+#define MANGLE(name) name
+#endif
+
```

Comment:
nit: unfortunate but we should `undef` at the end of the header as you did.

Reply:
Yeah, but I think this comment is also not applicable if you moved this to `macros.h`, right? I added this comment just in case you decided to keep the definition in `memory.h`.

Reply:
Sorry, just looked at the DIF and looks like you wanted to keep the definition here. SG.

Reply:
Ah, I didn't really think about moving it into macros.h.  I think if we need this kind of name mangling again, we should definitely dedupe the definition, but in the meantime I think it's nicer to define it closer to where we use it. It also gives us a nice place to hang the comment explaining the reason for name mangling.